interop/xds/custom_lb_test.go

```diff
@@ -64,14 +64,14 @@ func (s) TestCustomLB(t *testing.T) {
 				errCh.Send(errors.New("failed to receive metadata"))
 				return &testpb.SimpleResponse{}, nil
 			}
-			rpcBMD := md.Get(RPCBehaviorMetadataKey)
+			rpcBMD := md.Get("rpc-behavior")
 			if len(rpcBMD) != 1 {
-				errCh.Send(errors.New("only one value should be received for metadata key " + RPCBehaviorMetadataKey))
+				errCh.Send(errors.New("only one value should be received for metadata key " + "rpc-behavior"))
 				return &testpb.SimpleResponse{}, nil
 			}
 			wantVal := "error-code-0"
 			if rpcBMD[0] != wantVal {
-				errCh.Send(fmt.Errorf("metadata val for key \"%v\": got val %v, want val %v", RPCBehaviorMetadataKey, rpcBMD[0], wantVal))
+				errCh.Send(fmt.Errorf("metadata val for key \"%v\": got val %v, want val %v", "rpc-behavior", rpcBMD[0], wantVal))
```

Comment:
inline the rpc behavior string here as well (i.e. put it in the string rather than as an argument to formatting directive).